pkg/infra/teams/team1/social_capital.go

```diff
@@ -141,3 +143,85 @@ func (s *SocialAgent) updateSocialCapital(self agent.BaseAgent, fightDecisions d
 		s.socialCapital[key] = boundArray(s.socialCapital[key])
 	}
 }
+
+/**
+ * Tell other trusted agents above a threshold T about the A agents they admire the most,
+ * and the H agents they hate the most.
+ * Currently messages may be sent to recipients praising or denouncing the recipient
+ */
+func (r *SocialAgent) sendGossip(agent agent.BaseAgent) {
+	selfID := agent.Name()
+
+	sortedSCTrustworthiness := make([]SocialCapInfo, 0, len(r.socialCapital))
+	for k, sc := range r.socialCapital {
+		if k == selfID { // Exclude self
+			continue
+		}
+		sci := SocialCapInfo{ID: k, arr: sc}
+		sortedSCTrustworthiness = append(sortedSCTrustworthiness, sci)
+	}
+
+	sort.Slice(sortedSCTrustworthiness, func(i int, j int) bool {
+		return sortedSCTrustworthiness[i].arr[2] > sortedSCTrustworthiness[j].arr[2]
```

Comment:
I think it might be better to send sum(socialCapital[agentID]) rather than just the trustworthiness value, but it might not be? Maybe we should at least send the average value of trustworthiness and honour.

Reply:
Yes, maybe I should. I think I was following what the docs that @alexpondaven wrote, so we can also discuss this tomorrow